packages/testlab/src/shot.ts

```diff
@@ -8,6 +8,8 @@
  * https://github.com/hapijs/shot
  */
 
+// tslint:disable:no-any
+
 import {ServerRequest, ServerResponse} from 'http';
```

Comment:
Shouldn't this be `Request` / `Response` from `express` now?

Reply:
There are two ways how to use these Shot helpers:

1. To test LB4 code like a Sequence action. To do that, we need to mock Express `Request`/`Response` objects.

2. To test LB4 RestServer's request handler in isolation, without calling `http.createServer`. In this case, we need to mock core HTTP `ServerRequest`/`ServerResponse` objects.

That's why I am importing both Express and core HTTP types.